src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -57,13 +68,18 @@ private static IntPtr LoadNativeLibrary()
                     foreach (var searchDirectory in searchDirectories)
```

Comment:
This whole loop can be replace by using the TryLoad overload that takes DllImportSearchPath

Reply:
[DllImportSearchPath](https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/Runtime/InteropServices/DllImportSearchPath.cs) seems like a Windows specific enum and the overload takes one value. Can we it be used in this Unix case?

Reply:
`default` should work for the Unix case.

Reply:
Thanks. The whole `if` block is omitted for `netcoreapp`. The (no-throw) `TryLoad` overload that takes DllImportSearchPath required `assembly`, I have passed in the executing assembly.